staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -432,7 +442,7 @@ func NewKubectlCommand(o KubectlOptions) *cobra.Command {
 	registerCompletionFuncForGlobalFlags(cmds, f)
 
 	cmds.AddCommand(alpha)
-	cmds.AddCommand(cmdconfig.NewCmdConfig(clientcmd.NewDefaultPathOptions(), o.IOStreams))
+	cmds.AddCommand(cmdconfig.NewCmdConfig(f, o.IOStreams))
```

Comment:
In order to preserve compatibility with other commands, passing factory instead PathOptions makes sense. However, this looks like a behavioral change needs careful consideration and I think, it should be in totally separate PR prior to kuberc?

Reply:
Agreed should definitely be it's own PR.